src/data/scripts/campaign/colonies/VayraColonialExpeditionIntel.java

```diff
@@ -53,22 +55,19 @@ public enum KadurColonialExpeditionOutcome {
     protected MarketAPI target;
     protected MarketAPI from;
 
-    protected FactionAPI colonyFaction;
-
     protected boolean enteredSystem = false;
     protected KadurColonialExpeditionOutcome outcome;
 
     protected Random random = new Random();
 
-    @Override
-    public FactionAPI getFaction() {
-        return colonyFaction;
-    }
-
     @SuppressWarnings("OverridableMethodCallInConstructor")
```

Comment:
hah... well now that the problem has been *deleted* and resolved, this can stay i guess, otherwise i'd say "delete this and fix whatever it was suppressing" :D